assets/type_of_service.json

```diff
@@ -1,7 +1,8 @@
 [
   {
     "typeEn" : "SURF",
-    "typeNl" : "SURF"
+    "typeNl" : "SURF",
+    "typeHidden": true
```

Comment:
Looking at the name, at first I expected these TOS (type of service) to become hidden form fields. While studying the actual solution. I found out you solved this problem in a much more elegant way.

I think the property could be slightly more descriptive. Although it now correctly conveys the purpose. It however does suggest `input type = hidden` to me for some reason. But now, the suggestion for giving a more descriptive name.

`isDeferred`
`isSupplementary`

Feel free to leave as is!

Reply:
I understand your point.
I think it's difficult to find a more clear term.

The main reason for 'typeHidden', is that when you are updating `type_of_service.json`, `typeHidden` is the most clear option.
On first glance I would not know what to expect from options named isDeferred or isSupplementary.

So will leave as is.